### PR TITLE
New: Pretty Diff Component

### PR DESCRIPTION
**Pull request by vinteo**
image

### CODE REVIEW

File: src/components/alexandria/PrettyDiffComponent.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash';
+import DiffMatchPatch from 'diff-match-patch';
+import React, { PropTypes } from 'react';
+
+require('styles/alexandria/PrettyDiff.scss');
+
+const PrettyDiffComponent = ({ newText, oldText }) => {
```
Comment (etaoins) [inline src/components/alexandria/PrettyDiffComponent.js]: Very clean function. Love it. :100:

**Comment by etaoins** [comment]
LGTM! 🍏 🔴

**Comment by davepoon** [comment]
LGTM :thumbsup:

**Comment by vinteo** [comment]
@etaoins @davepoon merge?

**Pull request merged**